MinStack.java

```diff
@@ -0,0 +1,50 @@
+class MinStack {
+
+    /** initialize your data structure here. */
+    Stack<Integer> s;
```

Comment:
Good Job! Please add comments, time and space complexity. Also please add the other problem for this assignment.